pages/core-api/safe-installation-guide.mdx

```diff
@@ -0,0 +1,250 @@
+            Create a `.env` file in the root of the repository with the following content:
+            ```bash
+            MNEMONIC=funded_account_on_this_network
+            NODE_URL=RPC_endpoint_for_your_network
+            ```
+
+            Deploy the contracts by running this command:
+
+            ```bash
+            npm run deploy-all custom
+            ```
+        </Tabs.Tab>
+    </Tabs>
+
+4. The script should deploy all the singleton contracts (nine contracts in total).
+
+    Write down each address (example addresses for v1.3.0 could look like):
+    ```js
+    compatibility_fallback_handler: `0x017062a1dE2FE6b99BE3d9d37841FeD19F573804`
+    create_call: `0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d`
+    gnosis_safe: `0x69f4D1788e39c87893C980c06EdF4b7f686e2938`
+    gnosis_safe_l2: `0xfb1bffC9d739B8D520DaF37dF666da4C687191EA`
+    multi_send: `0x998739BFdAAdde7C933B942a68053933098f9EDa`
+    multi_send_call_only: `0xA1dabEF33b3B82c7814B6D82A79e50F4AC44102B`
+    proxy_factory: `0xC22834581EbC8527d974F8a1c97E1bEA4EF910BC`
+    sign_message_lib: `0x98FFBBF51bb33A056B08ddf711f289936AafF717`
+    simulate_tx_accessor: `0x727a77a074D1E6c4530e814F89E618a3298FC044`
+    ```
+
+
+### Record your contracts in the official registry
+
+You must share your singleton contract deployment addresses in the official public registry.
+1. Fork the [safe-deployments](https://github.com/safe-global/safe-deployments) GitHub repository.
+2. Add your chain ID to each of the nine JSON files in `src/assets/<version>`. If you deployed with the singleton deployment from above, you have to mark your chain's deployment as "canonical".
+For example, add this line to `gnosis_safe.json` to indicate the gnosis safe has the canonical address on your chain:
+    ```json
+    "<your_chain_id>": "canonical"
+    ```
+
+3. Open a pull request. Your pull request should follow this [example pull request](https://github.com/safe-global/safe-deployments/pull/679).
+
+</Steps>
+
+## Safe\{Core\} Infrastructure deployment
+
+Once you have deployed the Safe\{Core\} contracts, you must deploy the off-chain components of the Safe\{Core\} Stack, including backend and frontend services. 
+
+You have three options for deploying the Safe stack, depending on your needs.
+
+### Option 1: Safe Core Contributor Platform-as-a-Service
+
+We roll out new networks quarterly, depending on our internal capacity, and only for medium to large chains based on a strict scoring framework. 
+
+If you want to submit your chain for assessment in the next quarter, please fill out this [form](https://noteforms.com/forms/request-safe-ui-and-infra-support-4weugt).
+
+#### Hard requirements
+
+- EVM-compatible chain.
+- At least two dedicated RPC node providers, with preference for at least one Tier 1 provider of Infura, Alchemy, Quicknode. You can find more on RPC nodes [here](https://docs.safe.global/core-api/api-safe-transaction-service/rpc-requirements).
+
+### Option 2: Third-party integrators
```

Comment:
We should remove this section if we don't have data for this atm.

Reply:
I agree, but I will keep it there until after the call with Greg, if it is ok with you. Maybe they already have data.

Reply:
For now there will be an email link.